### ISSUE TITLE
It looks like VIRTUAL* is not enabled by default on old X11. According to the [Arch Linux Wiki](https://wiki.archlinux.org/index.php/Extreme_Multihead#VNC) you can enable VIRTUAL* by adding the following configuration for X11:

### ISSUE THREAD

**Issue by YaoLiMuMu** [kbumsik/VirtScreen#95]
It looks like VIRTUAL* is not enabled by default on old X11. According to the [Arch Linux Wiki](https://wiki.archlinux.org/index.php/Extreme_Multihead#VNC) you can enable VIRTUAL* by adding the following configuration for X11:

```
Section "Device"
    Identifier "intelgpu0"
    Driver "intel"
    Option "VirtualHeads" "1"
EndSection
```
Copy/paste the above code in `/etc/X11/xorg.conf.d/20-intel.conf` or `/usr/share/X11/xorg.conf.d/20-intel.conf`, and then reboot the system. This tells the Intel driver to create one virtual display, VIRTUAL1. You can change the number `"1"` in `Option "VirtualHeads" "1"` to any number to create VIRTUAL1, VIRTUAL2, ... and VIRTUALn (which is unnecessary for VirtScreen use case),

_Originally posted by @kbumsik in https://github.com/kbumsik/VirtScreen/issues/16#issuecomment-420711627_

**Comment by YaoLiMuMu** [comment]
Hi, I did so, but when I reboot ubuntu20 system, there is always a lot of screen tearing, like this

![image](https://user-images.githubusercontent.com/32826811/113478770-88377e00-94bd-11eb-94f9-29d50f8916b8.png)

**Comment by ductm208** [comment]
> Hi, I did so, but when I reboot ubuntu20 system, there is always a lot of screen tearing, like this
> 
> ![image](https://user-images.githubusercontent.com/32826811/113478770-88377e00-94bd-11eb-94f9-29d50f8916b8.png)

yes same to me, Lenovo P1 gen 3 dual cards (intel + nvidia quadro)

**Comment by pinkpigeonltd** [comment]
Linux Mint 20.2, Intel HD 520, Kernel 5.4.0-92 here

I am trying to follow the instructions [here](https://wiki.archlinux.org/title/Extreme_Multihead#VNC):

> 
> Using a virtual output
> 
> Some drivers, like the old "intel" driver, support virtual displays. In this case, create at-least one virtual display in xorg
> 
> /etc/X11/xorg.conf.d/20-intel.conf
> 
> Section "Device"
>     Identifier "intelgpu0"
>     Driver "intel"
>     Option "VirtualHeads" "1"
> EndSection
> 
> The name of the output will normally be VIRTUAL1 (you can check with xrandr without any argument). The virtual display can then be activated using.
> 
> $ xrandr --output VIRTUAL1 --mode<resolution>

I can see VIRTUAL1 when running 'xrandr', however, in the display manager, or other tools it's not shown. Xrandr shows it as 'disconnected'.

I tried 'connecting' it by running ```xrandr --output VIRTUAL1 --mode<resolution>```, but it just says 'cannot find mode'. How do I add a mode to a virtual display?

Edit: I tried adding a mode by using ```xrandr --addmode VIRTUAL1 1920x1080``` and then ```xrandr --output VIRTUAL1 --mode 1920x1080 --right-of eDP1``

But this just corrupts my screen and crashes Cinnamon.
